scripts/capture-raycast-configs.sh

```diff
@@ -58,20 +58,41 @@ EOF
 elif [[ "${1}" == 'i' ]]; then
   ! is_file "${target_file}" && error "Couldn't find file: '$(yellow "${target_file}")' for import operation; Aborting!!!"
 
+  # TODO: Get this from password manager
+  password=$RAYCAST_SETTINGS_PASSWORD
+
   open raycast://extensions/raycast/raycast/import-settings-data
 
-  # TODO: Need to get import working
   osascript <<EOF
     tell application "System Events"
-      key code 36
```

Comment:
since the `tell` has a corresponding `end tell` should the nested lines be indented? I'm not sure.